README.md

```diff
@@ -77,8 +77,12 @@ target is a url (http load tests) or host:port (grpc health test).  flags are:
   -grpc-port string
 	grpc server port. Can be in the form of host:port, ip:port or port.
 	(default "8079")
-  -grpc-secure
```

Comment:
we still need a way to standard TLS no?

Reply:
I'm not following - how do I get the previous behavior exactly ?

Reply:
The previous behavior is accomplished by using an https prefix with the grpc destination.
For example:

__grpc client__
```
fortio grpcping https://fortio.istio.io
```
__or any client that uses `GRPCRunnerOptions`__
```
fortio curl -loglevel debug http://localhost:8080/fortio/fetch/localhost:8080/fortio/?url=https://fortio.istio.io:443&load=Start&qps=-1&json=on&n=100&runner=grpc
```

Reply:
ic I missed that - does it work with fortio load -grpc https://... too ? (it should I guess as destination is passed down unchanged if I recall)

we should put tls/no tls in the (json) result object

we should probably document that behavior somewhere (that grpc   host:port is insecure unless a cert is given and https://host:port is secure)